src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.Impl.cs

```diff
@@ -48,19 +48,11 @@ internal struct Impl : IEquatable<Impl>
                 Z = new Vector2(m31, m32);
             }
 
+            private static readonly Impl _identity = new Impl () { X = Vector2.UnitX, Y = Vector2.UnitY, Z = Vector2.Zero };
+
             public static Impl Identity
```

Comment:
Could be written as one-liner too:
```suggestion
            public static Impl Identity { get; } = new Impl() { X = Vector2.UnitX, Y = Vector2.UnitY, Z = Vector2.Zero };
```